testdata/scripts/seed.txt

```diff
@@ -0,0 +1,36 @@
+# Check the binary with a given base64 encoded seed
+garble -debugdir=./debug -literals -seed=OQg9kACEECQ= build main.go
+exec ./main
+cmp stdout main.stdout
+grep \\x42\\x49\\x29\\x2f\\x5a\\x13\\xab\\xef\\xfa\\x26\\x33\\x3e\\x46\\xa9\\xec\\x3e\\x96\\xa3\\x73\\x15\\x7c\\xa2\\x38\\xf2\\xf0\\xaf\\x58\\x6a\\xac\\x2b\\x50\\x94\\x3b\\x5b\\x3e\\xda\\x2f\\xe7 $WORK/debug/main/z0.go
+grep zWuVH6jac $WORK/debug/main/z0.go
+grep \\xea\\x20\\xd0\\xe8\\x66\\x9c\\x72\\x24\\x9a\\x94\\xec\\x6a\\x8f\\x82\\xd3\\xdc $WORK/debug/main/z0.go    
+! binsubstr  main$exe 'teststring' 'teststringVar'
+
+[short] stop # checking that the build is reproducible and random is slow
+
+# Also check that the binary is reproducible.
+cp main$exe main_old$exe
+rm main$exe
+garble -literals -seed=OQg9kACEECQ= build main.go
+bincmp main$exe main_old$exe
+
+# Check the random binary
```

Comment:
you should also check that building a random binary twice doesn't produce the same binary twice

Reply:
Yes unfortunatly bincmp doesn't allow !bincmp

Reply:
Ah, we should be able to fix that, though.